### PR TITLE
Whatsapp implementation

### PR DESCRIPTION
**Pull request by JaqKent**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
 REACT_APP_ENDPOINT=https://wolf-backend-development.herokuapp.com/api/v1
+REACT_APP_WHATSAPP=https://api.whatsapp.com/send?phone=3815949119&text=Tienes%20una%20nueva%20Compra
```
Comment (nathan-hyan) [inline .env]: This file isn't supposed to be pushed, having sensible data it's dangerous. Add it to the .gitignore file and delete it from the repository (only from the repo, not from your local files... you still need it)

File: src/components/NavigationBar/index.tsx [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
-import Links from "./components/Links";
+import { animated, useTransition } from "react-spring";
 import menu from "assets/hamburgerIcon.svg";
 import cart from "assets/cartIcon.svg";
 import logo from "assets/logoNegro.png";
 
+import { ANIMATION_CONFIG } from "./constants";
+import Links from "./components/Links";
 import styles from "./styles.module.scss";
+import { CartContext } from "context/Cart";
```
Comment (nathan-hyan) [inline src/components/NavigationBar/index.tsx]: sort imports

File: src/components/NavigationBar/styles.module.scss [hunk 3]
```diff
@@ -24,4 +24,16 @@
...
+    position: absolute;
+    text-align: center;
+    width: 15px;
+    height: 15px;
+    color: $black;
+    font-size: 10px;
+    font-weight: $bold;
+    border-radius: 15px;
+    background-color: $primary;
+    margin-left: 5px;
```
Comment (nathan-hyan) [inline src/components/NavigationBar/styles.module.scss]: sort alphabetically

File: src/components/SearchInput/index.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  const [value, setValue] = useState("");
+  const navigate = useNavigate();
+
+  const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setValue(e.target.value);
+  };
+
+  const onSearch = (searchItem: string) => {
+    navigate(`/product/${searchItem}`);
+    console.log("search", searchItem);
```
Comment (nathan-hyan) [inline src/components/SearchInput/index.tsx]: Remove console.log()

File: src/components/SearchInput/styles.module.scss [hunk 5]
```diff
@@ -0,0 +1,59 @@
+@import "scss/colors";
+@import "scss/mixins";
+@import "scss/variable";
+
+.container {
+  @include flex(center, center, row);
+  background-color: $primary;
+  height: 60px;
+  min-width: 375px;
+  gap: 5px;
```
Comment (nathan-hyan) [inline src/components/SearchInput/styles.module.scss]: sort

File: src/components/SearchInput/styles.module.scss [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  &:empty {
+    border: none;
+  }
+}
+
+.dropdownRow {
+  cursor: pointer;
+  margin: 2px 0;
+  height: 30px;
+  text-align: center;
```
Comment (nathan-hyan) [inline src/components/SearchInput/styles.module.scss:42]: sort

File: src/components/SearchInput/styles.module.scss [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+  cursor: pointer;
+  &:hover {
+    background-color: darken($dark-navy, 2%);
+    color: $gray;
+    cursor: pointer;
+  }
+  &:active {
+    background-color: lighten($dark-navy, 8%);
+    transform: translateY(4px);
+  }
```
Comment (nathan-hyan) [inline src/components/SearchInput/styles.module.scss]: sort

File: src/screens/Cart/GratitudScreen/styles.module.scss [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+  .button {
+    @include flex(space-evenly, center, row);
+    background-color: $primary;
+    border-radius: 15px;
+    color: $white;
+    font-weight: bold;
+    height: 60px;
+    margin: 30px 0 30px 0;
+    width: 150px;
```
Comment (nathan-hyan) [inline src/screens/Cart/GratitudScreen/styles.module.scss:22]: sort

File: src/screens/Cart/index.tsx [hunk 9]
```diff
@@ -1,6 +1,7 @@
 import { useContext } from "react";
+import { useNavigate } from "react-router-dom";
 import { Controller, SubmitHandler, useForm } from "react-hook-form";
-import FormInput from "components/FormImput/FormImput";
+import FormInput from "components/FormInput";
 import NoItem from "./components/NoItem";
 import PurchaseList from "./components/PurchaseList";
 import { CartContext } from "context/Cart";
```
Comment (nathan-hyan) [inline src/screens/Cart/index.tsx]: Sort

File: src/screens/Cart/index.tsx [hunk 10]
```diff
@@ -15,6 +16,10 @@ function Cart() {
   const onSubmit: SubmitHandler<Info> = (info) => {
     checkout(info);
   };
+  const navigate = useNavigate();
+  /*  const handleGratitud = () => {
+    navigate(`/GratitudScreen`);
+  }; */
```
Comment (nathan-hyan) [inline src/screens/Cart/index.tsx]: Remove comment block

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Great job, some comments!

File: .env (hunk 1 above)
Comment (JaqKent) [reply .env]: done ...i think

File: src/components/NavigationBar/index.tsx (hunk 2 above)
Comment (JaqKent) [reply src/components/NavigationBar/index.tsx]: done

File: src/components/NavigationBar/styles.module.scss (hunk 3 above)
Comment (JaqKent) [reply src/components/NavigationBar/styles.module.scss]: done

File: src/components/SearchInput/index.tsx (hunk 4 above)
Comment (JaqKent) [reply src/components/SearchInput/index.tsx]: done

File: src/components/SearchInput/styles.module.scss (hunk 5 above)
Comment (JaqKent) [reply src/components/SearchInput/styles.module.scss]: done

File: src/components/SearchInput/styles.module.scss (hunk 6 above)
Comment (JaqKent) [reply src/components/SearchInput/styles.module.scss:42]: done

File: src/components/SearchInput/styles.module.scss (hunk 7 above)
Comment (JaqKent) [reply src/components/SearchInput/styles.module.scss]: done

File: src/screens/Cart/GratitudScreen/styles.module.scss (hunk 8 above)
Comment (JaqKent) [reply src/screens/Cart/GratitudScreen/styles.module.scss:22]: this is already sorting alphabetically

File: src/screens/Cart/index.tsx (hunk 9 above)
Comment (JaqKent) [reply src/screens/Cart/index.tsx]: done

File: src/screens/Cart/index.tsx (hunk 10 above)
Comment (JaqKent) [reply src/screens/Cart/index.tsx]: done

**Review by nathan-hyan — APPROVED** [review]

**Pull request merged**